functions/Export-DbaDacPackage.ps1

```diff
@@ -67,27 +67,27 @@ function Export-DbaDacPackage {
         https://dbatools.io/Export-DbaDacPackage
 
     .EXAMPLE
-        PS C:\> Export-DbaDacPackage -SqlInstance sql2016 -Database SharePoint_Config
+        PS C:\> Export-DbaDacPackage -SqlInstance sql2016 -Database SharePoint_Config -FilePath C:\SharePoint_Config.dacpac
 
-        Exports the dacpac for SharePoint_Config on sql2016 to $home\Documents\SharePoint_Config.dacpac
+        Exports the dacpac for SharePoint_Config on sql2016 to C:\SharePoint_Config.dacpac
 
     .EXAMPLE
         PS C:\> $options = New-DbaDacOption -Type Dacpac -Action Export
         PS C:\> $options.ExtractAllTableData = $true
         PS C:\> $options.CommandTimeout = 0
         PS C:\> Export-DbaDacPackage -SqlInstance sql2016 -Database DB1 -Options $options
 
-        Uses DacOption object to set the CommandTimeout to 0 then extracts the dacpac for DB1 on sql2016 to $home\Documents\DB1.dacpac including all table data.
+        Uses DacOption object to set the CommandTimeout to 0 then extracts the dacpac for DB1 on sql2016 to C:\Users\username\Documents\DbatoolsExport\ including all table data. The generated filename will contain the server name, database name, and a timestamp.
```

Comment:
I think it will make it more clear if we just put the file name that is generated based on the example script.

Reply:
ok, sounds good